pages/perfil/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { NextPage } from 'next'
+
+import ClientOnly from '@/views/Shared/ClientOnly'
+
+import ProfileContainers from 'containers/Perfil/ProfileContainers'
+
+const Profile: NextPage = () => {
+  return (
+    <ClientOnly>
+      <ProfileContainers />
+    </ClientOnly>
+  )
+}
```

Comment:
```suggestion
const Profile: NextPage = () => (
    <ClientOnly>
      <ProfileContainers />
    </ClientOnly>
  )
```

Note it still needs indentation fix